src/views/ReadArticle.js

```diff
@@ -0,0 +1,439 @@
+/* eslint-disable no-shadow */
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { withRouter } from "react-router-dom";
+import Navbar from "../components/common/AppBars/navBar";
+import Input from "../components/common/Inputs/TextInput";
+import {
+  fetchArticle,
+  deleteArticle
+} from "../redux/actions/readArticleActionCreator";
+import Button from "../components/common/Buttons/BasicButton";
+
+import {
+  isCurrentUserAuthor,
+  stringToHtmlElement,
+  calculateTimeStamp
+} from "../utils/helperFunctions";
+import MainArticle from "../components/common/Cards/main";
+import twitterIcon from "../assets/icons/twitter-icon.svg";
+import facebookIcon from "../assets/icons/fb-icon.svg";
+import thumbsUp from "../assets/img/like-icon.svg";
+import authorImage from "../assets/img/user.jpg";
+import dislikeIcon from "../assets/img/dislike-icon.svg";
+import heartIcon from "../assets/img/heart.svg";
+import bookmarkIcon from "../assets/img/bookmark-icons.svg";
+
+export const mapStateToProps = (state, ownProps) => ({
+  ...ownProps,
+  currentUser: state.login.currentUser,
+  asideArticles: state.fetchedArticle.asideArticles,
+  article: state.fetchedArticle
+});
+export const mapDispatchToProps = dispatch => ({
+  deleteOneArticle: slug => dispatch(deleteArticle(slug)),
+  fetchOneArticle: slug => dispatch(fetchArticle(slug))
+});
+
+export class Article extends Component {
+  constructor(props) {
```

Comment:
@YvesIraguha They no longer use the constructor. Please check this article -> [React Constructor](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599)